lib/ansible/modules/notification/typetalk.py

```diff
@@ -19,7 +19,7 @@
 version_added: "1.6"
 short_description: Send a message to typetalk
 description:
-  - Send a message to typetalk using typetalk API ( http://developers.typetalk.in/ )
+  - Send a message to typetalk using typetalk API ( https://developer.nulab-inc.com/docs/typetalk/ )
```

Comment:
Is there any relevance in having the URL in the module's description ?
I would move it into the notes and make it a proper link.
Use either U(https://developer.nulab-inc.com/docs/typetalk/) or L(Typetalk API,https://developer.nulab-inc.com/docs/typetalk/).